content/root/glossary.textile

```diff
@@ -0,0 +1,218 @@
+
+JSON Web Token (JWT) is an open standard for securely representing claims transmitted between parties. The claims and metadata of a JWT are in JSON format and can be verified by recipients because they are cryptographically signed. JWTs can be used for authenticating with Ably.
+
+See the "authentication documentation":/core-features/authentication#ably-jwt-process for further information.
+
+h2(#latency). Latency
+
+Latency is a measure of the delay between an operation being initiated and its effect. In the context of Ably, latency typically refers to the time elapsed between a message being published on a channel and being received by a channel subscriber on another connection.
+
+h2(#messages). Message
+
+A message is the unit of information sent using the Ably platform. Messages are sent by publishers on a channel and received by all subscribers to that channel.
+
+See the "messages documentation":/realtime/messages for further information.
+
+h2(#metachannels). Metachannel
+
+A metachannel is a channel that receives messages containing metadata or events relating to an application. Metachannels exist outside of regular channel namespaces and instead have their names qualified with @[meta]@, for example @[meta]log@ or @[meta]channel.lifecycle@.
+
+See the "metachannels documentation":/realtime/metachannels for further information.
+
+h2(#mqtt). MQTT
+
+Message Queuing Telemetry Transport (MQTT) is a lightweight publish-subscribe protocol for minimal code implementations, or low bandwidth networks.
+
+See "Wikipedia":https://en.wikipedia.org/wiki/MQTT for further information.
+
+h2(#mqtt-adapter). MQTT Adapter
+
+The Ably MQTT Adapter can translate between MQTT and the Ably protocol.
+
+See the "MQTT Adapter documentation":https://ably.com/documentation/mqtt for further information.
+
+h2(#persistence). Persistence
+
+Persistence (or persisted history) is a feature that enables messages on a channel to be stored on disk. The last message published on a channel can be stored for 365 days, or all messages on a channel can be stored for 24-72 hours. Persisted messages can be retrieved by API or replayed to clients using the "rewind":#rewind feature when they attach to a channel.
+
+See the "history documentation":/realtime/history for further information.
+
+h2(#presence). Presence
+
+The presence feature enables clients in a channel to be aware of other presence members in the same channel. Each client has a presence state, and that state is communicated to all presence subscribers on the channel when a client enters or leaves.
+
+See the "presence documentation":/realtime/presence for further information.
+
+h2(#pub-sub). Pub/Sub
+
+Publish-subscribe (or pub/sub) is a pattern of information exchange where a publisher does not need to interact directly with, or even know the existence or identities of, any subscribers. Conversely, a subscriber can gain access to published information without interacting directly with the publisher.
```

Comment:
This is quite an important topic so I think we should link to it in our docs for more explanation. This seems to be the topic that gives a quick overview of the concept: https://ably.com/documentation/core-features/channels. We should maybe also link to this page from the entries for "publisher" and "subscriber". Sorry, missed this on the first review.

Reply:
Ideally I'd like to incorporate that info into a cleaner introductory section and link to that. Agree it makes sense to link to the existing page for now though.

Reply:
Yes, totally agree, I was going to raise a ticket for it, but if you can do that please and pop it in your backlog for now that would be great. Thank you.